sql/data_node.sql

```diff
@@ -3,7 +3,7 @@
 -- LICENSE-APACHE for a copy of the license.
 
 -- Check if a data node is up
-CREATE OR REPLACE FUNCTION _timescaledb_internal.ping_data_node(node_name NAME) RETURNS BOOLEAN
+CREATE OR REPLACE FUNCTION _timescaledb_internal.ping_data_node(node_name NAME, timeout INTEGER = NULL) RETURNS BOOLEAN
```

Comment:
What's the unit here? Is it seconds or milliseconds?
I think it would be better to use an `Interval` type so that one can type, e.g., `4 seconds` or `4s`.

Another option is to use a timestamp to set a "deadline": `ping_data_node('dn1', now() + interval '4s')`